ember-bootstrap/addon/components/bs-dropdown/menu.js

```diff
@@ -99,7 +100,13 @@ export default class DropdownMenu extends Component {
   }
 
   get alignClass() {
-    return this.align !== 'left' ? `dropdown-menu-${this.align}` : undefined;
+    if (this.align === 'right') {
```

Comment:
We only need to render a CSS class if `@align` argument is `"right"`.

Bootstrap supports responsive alignment as well: https://getbootstrap.com/docs/5.3/components/dropdowns/#responsive-alignment But that was also not supported by Ember Bootstrap before. It would require multiple changes:

1. Allow `@align` to take multiple values, e.g. `["left", "lg-right"]`.
2. Add `data-bs-display` (BS5) or `data-display` attribute with value `"static"` to the `.dropdown-menu` element if responsive alignment is used.

Changing the assertion from `this.align !== 'left'` to `this.align === 'right'` does not drop any supported use case. It only ensures that we don't render an invalid CSS class for unsupported values `@align`.

Reply:
Should we add support for this? If so could you create an issue?

Reply:
I created https://github.com/ember-bootstrap/ember-bootstrap/issues/2121 to track it.